IMPLEMENTATION_PLAN_evals_recipe_ux_fixes.md

```
@@ -0,0 +1,30 @@
+
+## Stage 1: Auth And Recipe Load States
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)